### ISSUE TITLE
[css-cascade-6] :scope and featureless elements

### ISSUE THREAD

**Issue by andruud** [w3c/csswg-drafts#9025]
- With `@scope (:host)`, you can make the shadow host a scoping root.
- Since `:scope` is defined to match the scoping root, it would then be able to match that element, even if it's featureless.

_Or would it?_

Selectors:

> A [featureless](https://drafts.csswg.org/selectors/#featureless) element does not match any selector at all, except those it is **explicitly** defined to match (and [logical combination pseudo-classes](https://drafts.csswg.org/selectors/#logical-combination-pseudo-classes) representing those selectors).

Specs _appear_ to be saying that `@scope (:host) { :scope { ... } }` would match, except that they aren't **explicitly** saying this.

Should we state this explicitly?

@mirisuzanne @tabatkins

**Comment by tabatkins** [comment]
Yes, it needs to be explicitly stated. (And hm, Nesting probably needs to say that too...)

**Comment by andruud** [comment]
@astearns Proposed async resolution: `:scope can match the featureless shadow host when that host is the scoping element`.

**Comment by astearns** [comment]
RESOLVED: :scope can match the featureless shadow host when that host is the scoping element.
